pages/en/lb4/LoopBack-3.x.md

```diff
@@ -75,7 +77,38 @@ LoopBack v4 is moving to the well-known Model-(View-)Controller pattern, where t
 | Tooling               | loopback-cli and API Connect UI             | (TBA)            |
 
 
+## What's new and exciting in LoopBack 4
+
+LoopBack 4 is envisioned with new and exciting ideas highlighted but not limited to as follows. 
+
+- Leverage TypeScript for better code quality and productivity
+- Unify and simplify the asynchronous programming model/style around Promise and Async/Await
+- Implement an IoC Container with Dependency Injection for better visibility, extensibility and composability
+- Introduce Component as packaging model for extensions that can be plugged into LoopBack 4 applications 
+- Make everything else as components, such as REST, Authentication, and Authorization
+- Divide the responsibilities of LoopBack models into
+  - Controllers - to handle incoming API requests
+  - Repositories - to provide access to data stores
+  - Models - to define schemas for business objects
+  - Services - to interact with existing REST APIs, SOAP WebServices, and other form of services/microservices
+- Refactor the ORM into separate modules for different concerns 
+
 ## What's in the beta release
 
+The beta release is the first milestone of the LoopBack 4 journey. It's in no way meant to be functionally complete or ready for prime time. You should see it as a preview of what's coming. But we do have some building blocks starting to take shape, such as:
```

Comment:
Although it's not functionally complete or ready for production use, it provides a preview of what's coming, including: